package.json

```diff
@@ -7,7 +7,7 @@
     "start:client": "react-scripts start",
     "start:server": "json-server --watch db.json --port 3001",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
```

Comment:
@lpatmo what is this environment for and is it different from `jsdom`? If needed, maybe let's add it in our jest.config.js file instead so that all our configurations are in one place.

https://github.com/codebuddies/frontend/blob/0e4c12207ab61ba8d76399bdaa2cf3a4049094ff/jest.config.js#L135

Reply:
Ahh, good question! So this change is based on this error that I encoutered: https://github.com/testing-library/dom-testing-library/issues/477

Someone in the thread recommended the change above, and it wasn't until I added it that the tests passed. 

I experimented with changing `testEnvironment` to be `jest-environment-jsdom-sixteen`, but that failed. I'm not entirely sure if that was the right thing to add to jest.config.js -- can you help with that?

Reply:
Will check it out!

Reply:
Update here: @angelocordon made https://github.com/codebuddies/frontend/pull/106 which upped our jest version to 25+